### ISSUE TITLE
Laser definitions inconsistent

### ISSUE THREAD

**Issue by HighIander** [ComputationalRadiationPhysics/picongpu#3269]
The plane wave laser has a parameter called "ramp_init", in the laser.param comments it is referenced as "laser_init", which it says is twice the time of the time the laser is initialized before/after the maximum.
In the gaussian laser, the parameter is correctly named "laser_init", but its definition is different: without the "twice".
Its just a little inconvenience, adding to the possible confusion of newbees :-)

**Comment by PrometheusPi** [comment]
I agree - I was also confused today by this definition (despite being aware of it for years now) - The standard deviation is defined from the center of the pulse, but we multiply it by 0.5 before multiplying it with the init factor just as if the standard deviation would be a symmetrically defied as e.g. FWHM.

**Comment by BrianMarre** [comment]
@PrometheusPi is this issue still relevant in the new incidentField laser profiles?

**Comment by PrometheusPi** [comment]
@BrianMarre Yes this is still an open issue: 
`RAMP_INIT` is used e.g. in [PlaneWave](https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/include/picongpu/fields/incidentField/profiles/PlaneWave.def#L44) and `PULSE_INIT` (changed by @steindev) is used in e.g. [GaussianPulse](https://github.com/ComputationalRadiationPhysics/picongpu/blob/dev/include/picongpu/fields/incidentField/profiles/GaussianPulse.def#L65C39-L65C49)
